bucket/manictime.json

```
@@ -0,0 +1,26 @@
+{
+    "homepage": "https://www.manictime.com",
+    "description": "A time tracking software",
+    "version": "4.4.4",
+    "license": {
+        "identifier": "Proprietary",
+        "url": "https://www.manictime.com/pricing"
+    },
+    "url": "https://cdn.manictime.com/setup/v4_4_4_0/ManicTimeUsb.zip",
+    "hash": "80c60f51e1070dcdfb70daa7d5ae61b7085d932163a18a02fbb040946ba0bba8",
+    "shortcuts": [
+        [
+            "ManicTimeClient.exe",
+            "ManicTime"
+        ]
+    ],
+    "persist": "Data",
+    "checkver": {
+        "url": "https://www.manictime.com/Releases",
+        "regex": "ManicTime v([\\d.]+)"
```

```suggestion
        "jsonpath": "$.Version"
```

The endpoint returns XML in the browser but it will return JSON if called via Scoop